Makefile

```diff
@@ -133,7 +133,7 @@ endif
 	docker-compose run integration bundle exec rspec spec/components/$(COMPONENT)_spec.rb
 
 ## Experimental ##
-spec-ci:
+spec-ci: ci-env-vars
```

Comment:
is this needed?

Reply:
ah good spot - no it's not needed as it's being called on `setup-ci`. Will remove.